.github/workflows/cmake.yml

```diff
@@ -175,6 +175,12 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
+    - name: Information about platform
```

Comment:
I'm missing a verb in this name 😉 let's add e.g. "Gather"

Reply:
not done, and since you call the script "get system info" you can use "get" here as well 😉

Reply:
now done